src/components/core/GoogleAnalytics/GoogleAnalytics.jsx

```diff
@@ -18,10 +18,20 @@ class GoogleAnalytics extends React.Component {
 
   componentDidUpdate({ history }) {
     let { location: { pathname } } = history;
-    if (pathname.includes(Routes.MEMBER_DETAILS)) {
-      // In order to match the Member_Details route, we need to exclude memberId from the pathName
-      pathname = getMemberDetailsPathFromPathName(pathname);
+
+    // Here we are checking for route which have params append at it.
+    // If route contains a param then we remove the param. This for
+    // matching the path properly.
+    let isPathWithParam = false;

```

Comment:
Can be simplified to:
```suggestion
  // Define this in constant file.
    const urlListWithParams = ['MEMBER_DETAILS_WITH_PARAM', 'MEMBER_PROFILE_WITH_PARAM' ];
    
    if (urlListWithParams.includes(pathname)) {
      pathname = removeUrlParamsFromPathname(pathname);
    }

```

Reply:
As per discussion it is done.